src/components/AdminPage/AdminPage.scss

```diff
@@ -0,0 +1,52 @@
+@import '../../styles/mixins';
+
+.admin-page {
+  height: 100%;
+  width: 100%;
+  @include flex(column, flex-start, center);
+}
+
+.admin-title-container {
+  min-height: 20%;
+  width: 100%;
+  @include flex(row, center, center);
+}
+
+.admin-title {
+  font-size: 40px;
+}
+
+
+.card-container {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(5, 20%);
+  grid-template-rows: max-content;
+}
+
+.card {
+  justify-self: center;
+  @include flex(column, center, center);
+  border-radius: 5px;
+  height: 250px;
+  width: 250px;
+  margin: 20px;
+  background-color: lightgray;
+
+  &:hover {
+    cursor: pointer;
+    background-color: rgb(189, 189, 189);
+  }
+}
```

Comment:
These look so good! Thank you for building this and teaching me some other ways of responsive styling!